### DISCUSSION TITLE
Multiple Scripts 1 Container

### DISCUSSION THREAD

**Discussion by MoldyTaint** [tteck/Proxmox#1530] [Help Needed] [answered]
Normally if I'd want to run Radarr, Sonarr, Prowlarr I'd use

bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/sonarr.sh)"

bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/radarr.sh)"

bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/prowlarr.sh)"

If I want them all in 1 LXC instead, could I create the LXC Initially and then run these commands inside that sole LXC for the same results?

https://github.com/tteck/Proxmox/blob/main/install/sonarr-install.sh

https://github.com/tteck/Proxmox/blob/main/install/radarr-install.sh

https://github.com/tteck/Proxmox/blob/main/install/prowlarr-install.sh

**Comment by tteck** [comment, marked as answer]
Nope, it would result in multiple unbound variables.

**Reply by MoldyTaint** [reply]
Thank you. I will hopefully wait for the day a script comes out for them all together, until that time docker is gonna be there for me.

Thank you again for everything and such timely replies.
